Public/Src/Cache/ContentStore/Library/Service/LocalServerConfiguration.cs

```diff
@@ -108,7 +108,7 @@ public LocalServerConfiguration OverrideServiceConfiguration(ServiceConfiguratio
         public string GrpcPortFileName { get; set; } = DefaultFileName;
 
         /// <nodoc />
-        public int GrpcPoolSize { get; private set; } = 70;
+        public int? GrpcPoolSize { get; set; } = 70;

```

Comment:
I think we should remove all the '70' hardcoded in other places and just rely on the default constant.